BedrockServer.cpp

```diff
@@ -1481,6 +1420,9 @@ void BedrockServer::postPoll(fd_map& fdm, uint64_t& nextActivity) {
         }
         size_t count = BedrockCommand::getCommandCount();
         if (count) {
+            // For commands not initiated by a client (those with initiatingClientID = -1), we can have commands
+            // remaining here even with `CLIENTS_RESPONDED` being true. We may want to address this in the future so
+            // that we can't orphan these commands at shutdown.
```

Comment:
Is there an issue where I can read more about this?

Reply:
I just created one here: https://github.com/Expensify/Expensify/issues/222495